test/NumberConverterTest.java

```diff
@@ -0,0 +1,57 @@
+public class NumberConverterTest {
+
+    public static void main(String[] args) {
+        NumberConverterTest.inttostrTest();
+        NumberConverterTest.strtointTest();
+        NumberConverterTest.dbltostrTest();
+        NumberConverterTest.strtodblTest();
+
+    }
+
+    public static void inttostrTest() {
+
+        String result1 = NumberConverter.inttostr(15);
+        String result2 = NumberConverter.inttostr(-15);
+        if (("-15".equals(result2))) {
```

Comment:
you have almost the same code in 4 places, make a separate method

Reply:
add method